src/Input/TextField.js

```diff
@@ -7,6 +7,9 @@ import AutoComplete from '../AutoComplete/AutoComplete';
 import KeyCode from '../utils/KeyCode';
 import ThemeService from '../styles/ChamelThemeService';
 
+import IconButton from 'chamel/button/IconButton';
+import SearchIcon from 'chamel/icons/font/SearchIcon';
```

Comment:
We don't need to add search button in the text field component or any buttons. Text Field component should just display a text field.